.gitignore

```diff
@@ -1,24 +1,29 @@
-# Compiled class file
-*.class
+### IntelliJ IDEA ###
+out/
+!**/src/main/**/out/
+!**/src/test/**/out/
 
```

Comment:
Not sure, but shouldn't you also add  ./idea

Reply:
Yeah apparently I mistakenly added them will make it in order when I remove draft. PS the .idea/ in the .gitignore should be in your local .gitignore_global as they refer to your individual working environment instead of anything related to the project. So I will check this all! Thanks